pipelines.go

```diff
@@ -105,12 +105,10 @@ type PipelineTestSuites struct {
 
 // PipelineTestCases contains test cases details.
 type PipelineTestCases struct {
-	Status        string `json:"status"`
-	Name          string `json:"name"`
-	Classname     string `json:"classname"`
-	ExecutionTime int    `json:"execution_time"`
-	SystemOutput  string `json:"system_output"`
-	StackTrace    string `json:"stack_trace"`
```

Comment:
Why are you removing these two? The docs seem to mention they are actually returned.

Reply:
Netto change is removal of SystemOutput / StackTrace; this due that response could be nill, empty arry, or single object. and which one you get is unpredicateble. 

Wasn't able to come up with a good sollution

Reply:
OK... Then let's merge this for now and properly sort it later when someone needs the fields 👍🏻